opentreemap/treemap/templates/treemap/partials/form_buttons.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+<div class="col-xs-4 text-right">
+    <img class="spinner" src="{{ STATIC_URL }}img/spinner.gif" style="padding: 0 5px 5px 0; display: none;">
```

Comment:
There are many uses of the `spinner` class, and changing the SASS isn't compatible with all of them. This spinner needs some design attention anyway since it's on a dark background, so I thought the designer could sort it out.